src/helper.js

```diff
@@ -6,7 +6,8 @@ define(["qlik"], function (qlik) {
       'qlik-tabbed-container',
       'qlik-trellis-container',
       'filterpane',
-      'histogram'],
+      'histogram',
+      'sn-analysis-autochart'],
```

Comment:
```suggestion
      'sn-layout-container',
      'sn-analysis-autochart'],
```
@niekvanstaveren this as well right?

Reply:
Yes layout container cannot trellis. What about new container? Will that fall also under `container`?

Reply:
yea it will (I think). But can't hurt to add that as well.